gpytorch/variational/variational_strategy.py

```diff
@@ -12,18 +12,19 @@
     SumLinearOperator,
     TriangularLinearOperator,
 )
-from linear_operator.utils.cholesky import psd_safe_cholesky
```

Comment:
I think we keep this import instead of the changed one -- all of the linear algebra internals (gpytorch.utils, gpytorch.lazy) were separated out into their own package.

Reply:
Reverted